src/test/java/org/hobbit/core/components/TaskGeneratorTest.java

```diff
@@ -67,7 +67,7 @@
         // processing (max 100)
         testConfigs.add(new Object[] { 1, 10000, 100, 0 });
         // We use two data generators without parallel message processing
-        testConfigs.add(new Object[] { 2, 10000, 1, 0 });
+        testConfigs.add(new Object[] { 1, 10000, 1, 0 });
```

Comment:
Please revert this change as it changes the test configuration in a way that it does not fit to the description.